README.md

```diff
@@ -31,9 +32,9 @@ module "iam-config" {
   source          = "github.com/uptycslabs/terraform-aws-iam-config"
   resource_prefix = "cloudquery"
 
-  # Copy the AWS Account ID from Uptycs' UI
+  # Copy the Uptycs Account ID from Uptycs UI
   # Uptycs' UI : "Cloud"->"AWS"->"Integrations"->"ACCOUNT INTEGRATION"
-  aws_account_id = "Uptycs-AWS-ACCOUNT-ID"
+  upt_account_id = "Uptycs-ACCOUNT-ID"
```

Comment:
Why do we need to change this now?
If customer already copied these lines, this is going to cause an error as soon as they re-run/re-apply